docs/modules/user-guide/partials/proc_adding-a-command-group-to-a-devfile.adoc

```diff
@@ -0,0 +1,56 @@
+[id="proc_adding-a-command-group-to-a-devfile_{context}"]
```

Comment:
I'm not sure how it's done in devfiles docs but in OCP docs, we always include a comment near the beginning of the .adoc module file to indicate in what assembly the module is included. E.g.:

```
// Module included in the following assemblies:
//
// * monitoring/configuring-the-monitoring-stack.adoc
```

If needed, add to the other module files in this PR.

Reply:
We don't do this in our docs. But maybe we should, thanks for letting me know how OCP does it :) 

I'll discuss with the wider team, thanks!

Reply:
The mod docs standard removed this practice because maintaining the information requires effort and duplicates information that is already available by simply searching the repo for the file name. The mod docs team decided to drop the practice because we found the information was poorly maintained by writers making changes. There are also issues with cherry-picking a file across release branches that don't necessarily have the same files or file structure. 

The OpenShift Docs team had abandoned the practice, but then recently restored it in response to advocacy by some team members. Another team member is working on a script to automate updating maintaining this information. 

My advice is to not do it.

Reply:
Thanks, Rolfe, for the insight! I appreciate both of you telling me more about modular documentation--it all helps!